{{cookiecutter.profile_name}}/lsf_status.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+import time
+from typing import List
+from subprocess import CalledProcessError
+import sys
+from pathlib import Path
+if not __name__.startswith("tests.src."):
+    sys.path.append(str(Path(__file__).parent.absolute()))
+    from OSLayer import OSLayer
+else:
+    from .OSLayer import OSLayer
+
+
+class BjobsError(Exception):
+    pass
+
+
+class LSF_Status_Checker:
+    SUCCESS = "success"
+    RUNNING = "running"
+    FAILED = "failed"
+    STATUS_TABLE = {
+        "PEND": RUNNING,
+        "RUN": RUNNING,
+        "DONE": SUCCESS,
+        "PSUSP": RUNNING,
+        "USUSP": RUNNING,
+        "SSUSP": RUNNING,
+        "WAIT": RUNNING,
+        "EXIT": FAILED,
+        "POST_DONE": SUCCESS,
+        "POST_ERR": FAILED,
+        "UNKWN": RUNNING,
+    }
+
+    def __init__(self, jobid: int, outlog: str, WAIT_BETWEEN_TRIES: float = 5, TRY_TIMES: int = 3):
+        self._jobid = jobid
+        self._outlog = outlog
+        self.WAIT_BETWEEN_TRIES = WAIT_BETWEEN_TRIES
+        self.TRY_TIMES = TRY_TIMES
+
+    @property
+    def jobid(self):
```

Comment:
I know it's a pain, but could you please add return types?

Reply:
Done in https://github.com/leoisl/snakemake-lsf/commit/4eb7fe3e2c026e40c9e34d1a510759aca1f040cd